### DISCUSSION TITLE
Do we want to follow conventional commits for PR's?

### DISCUSSION THREAD

**Discussion by jdwilkin4** [jdwilkin4/contrast-ratio-repo#16] [Ideas]
I know that this is the first time many people are using git and working on a team.
I didn't know if we want to enforce conventional commits for PR names. 
https://www.conventionalcommits.org/en/v1.0.0/

**Comment by DBenMoshe** [comment]
It might be a good idea! I see this project working out, at least for me, as a course on how to use GitHub properly, and it would help me avoid developing bad habits

**Reply by Okibe00** [reply]
I think we should use conventional PR names, It helps to use what's common in the industry while learning.
